maths/series/geometric_series.py

```diff
@@ -10,11 +10,14 @@
 """
 
 
+from typing import Union
```

Comment:
```suggestion
from __future__ import annotations
```
And then use the new `|` syntax below.

Reply:
Alright